src/client/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import Navbar from './navbar/Navbar';
+import Slider from './slider/Slider';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Navbar />
+        <Slider />
```

Comment:
Do you want the slider to be in all components ? where you will call other components? I think we should restructure the front end files.